pkg/console/operator/sync_v400.go

```diff
@@ -116,6 +117,16 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 	// available is currently defined as "met the users intent"
 	if deploymentsub.IsReady(actualDeployment) && routesub.IsAdmitted(rt) {
 		co.ConditionDeploymentAvailable(operatorConfig)
+
+		if deploymentsub.IsReadyAndUpdated(actualDeployment) {
+			co.ConditionNotProgressing(operatorConfig)
+			version := os.Getenv("RELEASE_VERSION")
+			if co.versionGetter.GetVersions()["operator"] != version {
+				co.versionGetter.SetVersion("operator", version)
+			}
+		} else {
+			co.ConditionProgressing(operatorConfig)
```

Comment:
It looks like we won't report progressing until at least one replica is ready?

We might need to pull this outside of the `if deploymentsub.IsReady(actualDeployment) && routesub.IsAdmitted(rt) {` block

Reply:
Do we need to remove or update the lines above that deal with progressing? (lines 111-114)

Reply:
Yeah, I don't think that nesting is quite right.  We can be progressing no matter how many pods we have.  As far as I understand, we are progressing until anything we manage stops changing.

Reply:
True, mainly wanted to see by the additional commit if jenskins will error out, but seems we are on the right path